lib/migrations/6-default-steps.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn(
+      'event',
+      'default_steps',
+      {
+        type: Sequelize.INTEGER
+      }
```

Comment:
No need for this block, you should be able to just put `Sequelize.INTEGER` on line 6.